webpack/scenes/ContentViews/details/DetailsContainer.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector, shallowEqual } from 'react-redux';
+import PropTypes from 'prop-types';
+import { STATUS } from 'foremanReact/constants';
+import EmptyStateMessage from '../components/EmptyStateMessage';
+import Loading from '../components/Loading';
+import getContentViewDetails from './ContentViewDetailActions';
+import { selectCVDetails,
+  selectCVDetailStatus,
+  selectCVDetailError } from './ContentViewDetailSelectors';
+
+const DetailsContainer = ({ children, cvId, isOpen }) => {
+  const dispatch = useDispatch();
+  const details = useSelector(state => selectCVDetails(state, cvId), shallowEqual);
+  const status = useSelector(state => selectCVDetailStatus(state, cvId), shallowEqual);
+  const error = useSelector(state => selectCVDetailError(state, cvId), shallowEqual);
+
+  useEffect(() => {
+    if (isOpen && Object.keys(details).length === 0) {
+      dispatch(getContentViewDetails(cvId));
+    }
+  });
```

Comment:
Without a dependency array, wouldn't this repeatedly make API requests on every render until the first one comes back?